src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/GetPrintableDocumentsForEvent.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object GetPrintableDocumentsForEvent extends PerformanceTestsConfig {
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("getPrintableDocumentsForEvent")
```

Comment:
rename `getPrintableDocumentsForEvent` to `getCasePrintableDocuments` please

Reply:
done